packages/clerk-js/src/ui/components/KeylessPrompt/index.tsx

```diff
@@ -444,9 +464,34 @@ const _KeylessPrompt = (_props: KeylessPromptProps) => {
             }
           `}
         >
-          {isLoading ? <Spinner size={'sm'} /> : <> {claimed ? 'Get API keys' : 'Claim keys'}</>}
-        </button>
+          {claimed ? 'Get API keys' : 'Claim keys'}
+        </a>
       </Flex>
+      <BodyPortal>
+        <a
+          href={`#${buttonIdentifier}`}
+          css={css`
+            position: fixed;
+            left: -999px;
+            top: 1rem;
+            z-index: 999999;
+            border-radius: 0.375rem;
+            background-color: white;
+            padding: 0.25rem 0.5rem;
+            font-size: 0.875rem;
+            color: black;
+            text-decoration: underline;
+
+            &:focus {
+              left: 1rem;
+              outline: 2px solid;
+              outline-offset: 2px;
+            }
+          `}
+        >
+          Skip to keyless mode content
```

Comment:
Since this is specific to Clerk 👀 
```suggestion
          Skip to Clerk keyless mode content
```

Reply:
Good call!